extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -622,13 +625,23 @@ export class AmpStoryPage extends AMP.BaseElement {
           const {height, width} = layoutBox;
           state.height = height;
           state.width = width;
+          state.vh = height / 100;
         },
         mutate: (state) => {
           const {height, width} = state;
           if (state.height === 0 && state.width === 0) {
             return;
           }
           this.storeService_.dispatch(Action.SET_PAGE_SIZE, {height, width});
+          if (!this.cssVariablesStyleEl_) {
+            const doc = this.win.document;
+            this.cssVariablesStyleEl_ = doc.createElement('style');
+            this.cssVariablesStyleEl_.setAttribute('type', 'text/css');
+            doc.head.appendChild(this.cssVariablesStyleEl_);
+          }
+          this.cssVariablesStyleEl_.textContent = `:root {--story-page-vh: ${px(
```

Comment:
Nit: Why isn't this just an inline style on the `<html>` element?

Reply:
Specifically for this PR it's because[ it used to be like that](https://github.com/ampproject/amphtml/blob/4e2ac0cfeff422a83b7348d82319928e5d986691/extensions/amp-story/1.0/amp-story-page.js#L646), so I reverted that back.

In general, I think inlining the style would have made more sense (+ you don't have to keep track of this extra element).